core/common/src/main/java/alluxio/AlluxioConfiguration.java

```diff
@@ -43,6 +42,26 @@
    */
   String get(PropertyKey key, ConfigurationValueOptions options);
 
+  /**
+   * @param key the key to get the value for
+   * @param defaultValue the value to return if no value is set for the specified key
+   * @return the value
+   */
+  default String getOrDefault(PropertyKey key, String defaultValue) {
+    return containsKey(key) ? get(key) : defaultValue;
```

Comment:
I think the name `containsKey` is really confusing now. Can we deprecate it and use something like `hasValueSet` instead?

Reply:
good idea, deprecated and changed the name to `isSet` to be short and similar to Thrift